workshops/models.py

```diff
@@ -478,6 +482,24 @@ def __str__(self):
 
 #------------------------------------------------------------
 
+class Language(models.Model):
+    """A language tag.
+
+    https://tools.ietf.org/html/rfc5646
+    """
+    name = models.CharField(
+        max_length=STR_MED,
+        help_text='Description of this language tag in English')
+    subtag = models.CharField(
+        max_length=STR_SHORT,
+        help_text=
+            'Primary language subtag.  '
```

Comment:
The IANA calls them "subtags" and not tags.
See <http://www.iana.org/assignments/language-subtag-registry/language-subtag-registry>